SpectraLogic.EscapePodClient/Model/EscapePodJob.cs

```diff
@@ -89,7 +89,7 @@ public class EscapePodJob : IEscapePodJob
         /// <param name="created">The created.</param>
         /// <param name="progress">The progress.</param>
         /// <param name="status">The status.</param>
-        public EscapePodJob(EscapePodJobId jobId, EscapePodJobType jobType, int numberOfFiles, long totalSizeInBytes, string created, double progress, EscapePodJobStatus status)
+        public EscapePodJob(string jobId, EscapePodJobType jobType, int numberOfFiles, long totalSizeInBytes, string created, double progress, EscapePodJobStatus status)
```

Comment:
The jobId will be a Guid, so you could use that type instead of a string for stronger typing.